### PR TITLE
✨ Add Keplr Sign-In button on top of Authcore Sign-In button

### PR DESCRIPTION
**Pull request by shellyYG**

### CODE REVIEW

File: util/cosmos/iscn/sign.js [hunk 1]
```diff
@@ -56,17 +56,26 @@ function getPublisherISCNPayload(user, { publisher, license }) {
...
-        rewardProportion: 1,
-        contributionType: 'http://schema.org/author',
-      });
-      break;
     }
   }
+  if (userId && displayName) {
+    stakeholders.unshift({
+      entity: {
+        '@id': userId,
```
Comment (williamchong) [inline util/cosmos/iscn/sign.js:64]:
```suggestion
        '@id': `${EXTERNAL_URL}/${userId}`,
```

File: pages/in/widget/iscn/index.vue [hunk 2]
```diff
@@ -379,6 +395,8 @@ export default {
       'fetchCurrentCosmosWallet',
       'prepareCosmosTxSigner',
       'calculateISCNTxTotalFee',
+      'loginUser',
+      'loginByCosmosWallet',
```
Comment (williamchong) [inline pages/in/widget/iscn/index.vue:399]: Remove if not used

File: store/modules/actions/payment.js [hunk 3]
```diff
@@ -49,7 +50,7 @@ export async function sendCosmosPayment(
         memo,
       }, signer));
     }
-    if (metadata) await api.apiPostTxMetadata(txHash, metadata);
+    if (metadata && !isNonLikerKeplrConnected) await api.apiPostTxMetadata(txHash, metadata);
```
Comment (williamchong) [inline store/modules/actions/payment.js:53]: Note: Maybe we should allow submitting metadata via a signature from the sender address later

File: util/cosmos/iscn/constant.js [hunk 4]
```diff
@@ -7,12 +7,20 @@ export const ISCN_PUBLISHERS = {
...
+  },
 };
 
 export const ISCN_LICENSES = {
   matters: {
     '/': 'QmRvpQiiLA8ttSLAXEd5RArmXeG4qWEsKPmrB7KeiLSuE4',
   },
+  wordpress: {
+    '/': 'QmZhRNkZaSnhDr6gBC22zwhTjsGyUx39tm8gjFYnTr2SjN',
+  },
```
Comment (williamchong) [inline util/cosmos/iscn/constant.js:23]: Are these new fields used in this PR?

File: util/cosmos/iscn/sign.js [hunk 5]
```diff
@@ -3,7 +3,6 @@ import BigNumber from 'bignumber.js';
 import { ISCNSigningClient } from '@likecoin/iscn-js';
 
 import { ISCN_RPC_URL, ISCN_PUBLISHERS, ISCN_LICENSES } from './constant';
-import { EXTERNAL_URL } from '../../../constant';
```
Comment (williamchong) [inline util/cosmos/iscn/sign.js:6]: Please keep `EXTERNAL_URL`

**Review by williamchong — CHANGES_REQUESTED** [review]

File: util/cosmos/iscn/constant.js (hunk 4 above)
Comment (shellyYG) [reply util/cosmos/iscn/constant.js:23]: No, forgot to remove :P

**Review by williamchong — APPROVED** [review]

**Pull request merged**